### PR TITLE
Add Script Command: Generate Password

### PR DESCRIPTION
**Pull request by hofmannsven**
This script generates a random password with 40 chars and copies it to the clipboard.

Example: LsK9XN5CYq4YHFTR4gADCfne/Jtn/I+AbHoUeLJo

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -649,6 +649,22 @@
...
+        "description" : "Generates a random password and copies it to the clipboard.",
+        "filename" : "generate-password.sh",
+        "icon" : "💻",
+        "mode" : "silent",
+        "needsConfirmation" : null,
+        "packageName" : "Developer Utils",
+        "refreshTime" : null,
+        "schemaVersion" : 1,
+        "title" : "Generate Password"
+      },
```
Comment (unnamedd) [inline extensions.json:667]: The `extensions.json` and `extension.md` are generated automatically, any changes made here will be overwritten.

File: extensions.md [hunk 2]
```diff
@@ -77,6 +77,7 @@ This repository contains sample commands and documentation to write your own one
 | 💻 | [Decode Base64](developer-utils/decode-base64.sh) | N/A | Raycast |
 | <img src="https://raw.githubusercontent.com/raycast/script-commands/master/developer-utils/images/jwt-logo.png?raw=true" width="20" height="20"> | [Decode JWT](developer-utils/decode-jwt.sh) | N/A | Raycast |
 | 💻 | [Encode Base64](developer-utils/encode-base64.sh) | N/A | Raycast |
+| 💻 | [Generate Password](developer-utils/generate-password.sh) | Generates a random password and copies it to the clipboard. | [Sven Hofmann](https://github.com/hofmannsven) |
```
Comment (unnamedd) [inline extensions.md:80]: The rollback of this addition is recommended, it is not needed.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Amazing addition! Just need to remove the changes on the files mentioned in the review comments.

Thanks for the Script Command and welcome aboard, @hofmannsven!

File: developer-utils/generate-password.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Generate Password
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.author Sven Hofmann
+# @raycast.authorURL https://github.com/hofmannsven
+# @raycast.description Generates a random password and copies it to the clipboard.
+# @raycast.icon 💻
```
Comment (unnamedd) [inline developer-utils/generate-password.sh:12]: Maybe this icon 🔑 here might be better. What do you think?

Comment (hofmannsven) [reply developer-utils/generate-password.sh]: I changed the icon from 💻 to 🔑

**Review by unnamedd — APPROVED** [review]

**Review by PitNikola — CHANGES_REQUESTED** [review]
Nice one, thanks for contribution!

I suppose user should have gpg installed? Let's mention it in the comments section and make this file `.template.` as per [new guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-additional-modification).

Sorry, we didn't have guidelines before 😬

**Comment by crstauf** [comment]
> make this file `.template.` as per new guidelines

@PitNikola To me the contributions do not state that `.template.` is required for dependencies. Is this really the case? If so, `curl` and `open` are technically dependencies.

**Comment by unnamedd** [comment]
> Do scripts with dependencies really need to have `.template.`? If so, `curl` and `open` are technically dependencies.

`open` is part of the system, `curl` is almost part of the system. It is embedded in almost every operating system, for that reason, I don't see reason to have them as `.template.`. To me it's different when you need to install something else, like the `gpg` here.

**Comment by crstauf** [comment]
> when you need to install something else

A template is a starting point for creating something, and does not speak to other requirements. 

In painting, you've got a template for painting symbols, but that doesn't speak at all to the type or color or possession or quality of the paint.

**Comment by PitNikola** [comment]
Okay, it's a valid point, `.template.` indeed doesn't speak to dependencies. Perhaps having a requirement to handle missing dependency is enough + instruction at the top in comments section stating the dependency.

**Comment by PitNikola** [comment]
Okay, @hofmannsven, for now let's do it this way:

- Add handling of non-installed gpg command line ([here's example](https://github.com/raycast/script-commands/pull/67/files))
- Mention it in the comments section, e.g.

```
#!/bin/bash

# Dependencies: Requires installed [gpg cli](https://formulae.brew.sh/formula/gnupg)

# Required parameters:
# @raycast.schemaVersion 1
# @raycast.title Generate Password
# @raycast.mode silent

# Optional parameters:
# @raycast.author Sven Hofmann
# @raycast.authorURL https://github.com/hofmannsven
# @raycast.description Generates a random password and copies it to the clipboard.
# @raycast.icon 🔐
```

**Comment by PitNikola** [comment]
I've added new section to [CONTRIBUTING.md](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md) regarding to dependencies.

**Comment by thomaspaulmann** [comment]
Hey @hofmannsven, looks like your script isn't in the right folder. We restructured our repository and you should move the script to `commands/developer-utils` as mentioned in our guidelines. Sorry for the inconvenience and thanks for understanding.

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
As mentioned, please move your script.

**Review by thomaspaulmann — APPROVED** [review]
Perfect and congrats for your first contribution @hofmannsven. Off to many more :-)

**Pull request merged**